botbuilder-instumentation/Instumentation/BotFrameworkInstrumentation.cs

```diff
@@ -131,6 +132,19 @@ public void TrackQnaEvent(IActivity activity, string userQuery, string kbQuestio
             }
         }
 
+        public void TrackGoalTriggeredEvent(IActivity activity, string goalName,
+            IDictionary<string, string> goalTriggeredEventProperties = null, string eventName = TelemetryEventTypes.GoalTriggeredEvent)
```

Comment:
@garypretty Since this is a custom made method dedicated only for tracking goals, there is no need to pass the parameter `eventName` or to initialize it. Instead we can just set the event name directly in line 144 as 'TelemetryEventTypes.GoalTriggeredEvent', much like it was implemented in the `TrackQnaEvent` method.
Otherwise, if someone passes another name for the event, your ibex template will not pick it up.